infra/los/templates/los_database.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "los.fullname" . }}-los-database
+  namespace: {{ .Values.los_namespace }}
+  labels:
+    {{- include "los.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "los.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        app: los-database
+        {{- include "los.labels" . | nindent 8 }}
+        {{- with .Values.podLabels }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}-los-database
+          image: "{{ .Values.los_database.image.repository }}:{{ .Values.los_database.image.tag }}"
+          imagePullPolicy: {{ .Values.los_database.image.pullPolicy }}
+          env:
+            - name: POSTGRES_HOST_AUTH_METHOD
+              value: trust
+            - name: POSTGRES_DB
+              value: {{ .Values.los_database.db_name }}
+            - name: POSTGRES_USER
+              valueFrom:
+                secretKeyRef:
+                  name: backend-secrets
+                  key: database-user
+            - name: POSTGRES_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: backend-secrets
+                  key: database-password
```

Comment:
Could you use `envFrom` here? Works for secrets, too.
https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#configure-all-key-value-pairs-in-a-configmap-as-container-environment-variables

Reply:
Resolved